x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/EvaluatorImplementer.java

```diff
@@ -54,11 +60,14 @@ public class EvaluatorImplementer {
     private final ProcessFunction processFunction;
     private final ClassName implementation;
 
+    private final boolean mv;
```

Comment:
Probably better to call this `processOutputsMultivalue` or something long and descriptive like that. It's a java tradition!